lang/cs/Org.Apache.REEF.Tang/Formats/ConfigurationFile.cs

```diff
@@ -223,6 +223,29 @@ private static void AddConfiguration(IConfigurationBuilder conf, StreamReader re
             return property;
         }
 
+        public static void ProcessConfigData(IConfigurationBuilder conf, IList<KeyValuePair<string, string>> settings, string language)
```

Comment:
This is a new API, right? So we should be able to make it `internal` from the get-go.